source/Calamari.Azure/Calamari.Azure.csproj

```diff
@@ -105,4 +106,8 @@
     <Copy SourceFiles="@(FSharpFiles)" DestinationFolder="$(PublishDir)/FSharp/" />
     <Copy SourceFiles="@(ScriptCSFiles)" DestinationFolder="$(PublishDir)/ScriptCS/" />
   </Target>
+  <Target Condition="$([MSBuild]::IsOSUnixLike())" Name="Build">
+    <Message Text="This build is unsupported on unix like environments"/>
+  </Target>
+  <Import Condition="!$([MSBuild]::IsOSUnixLike())" Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
```

Comment:
I managed to get this working by moving from implicit to explicit sdk imports: https://docs.microsoft.com/en-us/visualstudio/msbuild/how-to-use-project-sdk?view=vs-2019 This gave me something that I could then conditionally skip.

Reply:
Nicely done :+1: